cio/io.go

```diff
@@ -24,11 +24,12 @@ type Config struct {
 type IO interface {
 	// Config returns the IO configuration.
 	Config() Config
-	// Cancel aborts all current io operations
+	// Cancel aborts all current io operations.
 	Cancel()
-	// Wait blocks until all io copy operations have completed
+	// Wait blocks until all io copy operations have completed.
 	Wait()
-	// Close cleans up all open io resources
+	// Close cleans up all open io resources. Cancel() will always be called before
```

Comment:
It reads weird somehow.

Maybe simply `Cancel() will always be called before closing the IOs`?

Reply:
I changed the wording a bit, let me know if it's better